spec.md

```diff
@@ -8,66 +8,96 @@ title: Semantic Specification
 
 
 ~~~
-Parentage/causal relationships for spans from a single trace
+Causal relationships for Spans in a single trace
 
-        [Span A]  <--(the root span)
+        [Span A]  ←←←(the root span)
             |
      +------+------+
      |             |
- [Span B]      [Span C]
+ [Span B]      [Span C] ←←←(Span A is Span C's parent)
      |             |
- [Span D]      +---+-----+
-               |         |
-           [Span E]  [Span F]
-                         |
-                +--------+--------+
-                |        |        |
-            [Span G] [Span H] [Span I]
+ [Span D]      +---+-------+
+               |           |
+           [Span E]    [Span F] >>> [Span G] >>> [Span H]
```

Comment:
Can the sequence predecessor reference also fan out and have multiple children? I guess it could.